tests/Provide/Router/CliRouterTest.php

```diff
@@ -100,4 +100,14 @@ public function testError()
         ];
         $this->router->match($globals, []);
     }
+
+    public function testStdInCleanup()
+    {
+        file_put_contents($this->stdInFile, '');
+        $exists = file_exists($this->stdInFile);
+        $this->assertTrue($exists);
+        unset($this->router);
```

Comment:
@koriym 
Thank you for the advice! I see.
Both case works, so I'll go with this (`unset`) code.